src/core/server/environment/resolve_uuid.test.ts

```diff
@@ -91,7 +91,7 @@ describe('resolveInstanceUuid', () => {
         expect(logger.debug).toHaveBeenCalledTimes(1);
         expect(logger.debug.mock.calls[0]).toMatchInlineSnapshot(`
           Array [
-            "Updating Kibana instance UUID to: CONFIG_UUID (was: FILE_UUID)",
+            "Updating Kibana instance UUID to: ${DEFAULT_CONFIG_UUID} (was: ${DEFAULT_FILE_UUID})",
```

Comment:
NIT: I know this was meant for the best, but please avoid using string interpolations in inline snapshots unless strictly necessary (such as when asserting against paths that will not match between local and CI), as those snapshots will be overridden when updated via `-u` anyway.

Reply:
Is it better to embed the values into the snapshot string then?

Reply:
Ah! I didn't realize `-u` handles inline as well. Makes sense. Will update to static strings.